### PR TITLE
Authorization

### PR DESCRIPTION
**Pull request by Hamza-Heidar**
- Applicants can't access recruiter pages
- Recruiters can't access applicant pages

**This pull request have completed these problems:**
- Add your completed task here!

### CODE REVIEW

File: src/main/java/kth/iv1201/recruitment/config/SecurityConfig.java [hunk 1]
```diff
@@ -28,8 +28,11 @@
 	 */
 	@Override
 	protected void configure(HttpSecurity http) throws Exception {
-		http.authorizeRequests().antMatchers("/applicants/**").authenticated().and().formLogin().loginPage(
-				"/login").failureUrl("/login-error").permitAll().and().logout().logoutUrl("/logout").clearAuthentication(true).deleteCookies().permitAll();
+		http.authorizeRequests()
+				.antMatchers("/applicants/**").access("hasRole('ROLE_recruiter')")
+				.antMatchers("/positions").access("hasRole('ROLE_applicant')")
```
Comment (arifjehoh) [inline src/main/java/kth/iv1201/recruitment/config/SecurityConfig.java:33]: Remove the empty space.

File: src/main/java/kth/iv1201/recruitment/controller/WebController.java [hunk 2]
```diff
@@ -139,4 +139,14 @@ public String summary(@PathVariable int id, Model model) {
...
 	}
+
+	/**
+	 * Returns the page where applicants can see available positions
+	 * @return
+	 */
+	@RequestMapping(value = "/positions")
+	public String positions() {
+
+		return "/positions";
```
Comment (arifjehoh) [inline src/main/java/kth/iv1201/recruitment/controller/WebController.java:150]: Create `private static final String` variable for position route.

File: src/main/resources/templates/positions.html [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+<div th:include="navbar"/>
+<h1> Here you can apply for available positions </h1>
+<div>
+
+
+
+No available positions
+
+
+</div>
```
Comment (arifjehoh) [inline src/main/resources/templates/positions.html:20]: Remove the empty spaces.

**Review by arifjehoh — CHANGES_REQUESTED** [review]
Fix this, then we can proceed on reviewing.

**Comment by arifjehoh** [comment]
@Hamza-Heidar Make sure to remember to use the prefixes in commits.

**Review by mrporsev — APPROVED** [review]

**Pull request merged**